bedrock/firefox/templates/firefox/new/scene2.html

```diff
@@ -29,29 +29,18 @@
 
 {% block content %}
 <main role="main" id="stage">
-  <div class="horizon">
+  <div class="main-container">
+    <div class="horizon">
     <div class="stars">
       <div class="inner-container">
         <header id="masthead">
           <div id="tabzilla">
             <a href="{{ url('mozorg.home') }}" data-link-type="nav" data-link-name="tabzilla">Mozilla</a>
           </div>
-          <h2>
-          {% if LANG == 'en-US' %}
-            <a href="{{ url('firefox') }}">{{ high_res_img('firefox/template/header-logo-inverse.png', {'alt': 'Firefox', 'width': '185', 'height': '70'}) }}</a>
-          {% else %}
-            {{ high_res_img('firefox/template/header-logo-inverse.png', {'alt': 'Firefox', 'width': '185', 'height': '70'}) }}
-          {% endif %}
-          </h2>
+          <h2>{{ high_res_img('firefox/template/header-logo-inverse.png', {'alt': 'Firefox', 'width': '185', 'height': '70'}) }}</h2>
```

Comment:
Same question on dropping the link. Just making sure it's intended.

Reply:
Will update this one as well